docs/message_retention_policies.md

```diff
@@ -136,24 +136,34 @@ the server's database.
 
 ### Lifetime limits
 
-**Note: this feature is mainly useful within a closed federation or on
-servers that don't federate, because there currently is no way to
-enforce these limits in an open federation.**
-
-Server admins can restrict the values their local users are allowed to
-use for both `min_lifetime` and `max_lifetime`. These limits can be
-defined as such in the `retention` section of the configuration file:
+Server admins can set limits on the values of `max_lifetime` to use when
+purging old events in a room. These limits can be defined as such in the
+`retention` section of the configuration file:
 
 ```yaml
   allowed_lifetime_min: 1d
   allowed_lifetime_max: 1y
 ```
 
-Here, `allowed_lifetime_min` is the lowest value a local user can set
-for both `min_lifetime` and `max_lifetime`, and `allowed_lifetime_max`
-is the highest value. Both parameters are optional (e.g. setting
-`allowed_lifetime_min` but not `allowed_lifetime_max` only enforces a
-minimum and no maximum).
+These values are evaluated when running purge jobs, to adjust the
+effective value of `max_lifetime`, and make sure it's within the
+`[allowed_lifetime_min, allowed_lifetime_max]` range. If the value of
```

Comment:
```suggestion
`[allowed_lifetime_min, allowed_lifetime_max]` range, inclusive. If the value of
```

Reply:
The writing `[a, b]` already imply it's inclusive. An exclusive version would look like `]a, b[` (fully exclusive) or `[a, b[` (partially exclusive). See https://en.wikipedia.org/wiki/Interval_(mathematics)#Including_or_excluding_endpoints

Reply:
I know, but server admins may not understand mathematical notation is all.

Reply:
I'll probably rephrase it so it uses plain English instead of a weird mix of both then.

Reply:
Done in https://github.com/matrix-org/synapse/pull/8529/commits/ddd1f2bc9af1130e5ce209eb44f4c3c168115f19